java_src/foam/dao/index/AltIndex.java

```diff
@@ -1,96 +0,0 @@
-/**
```

Comment:
I assume it's generated by the `src/foam/dao/index/*.js` files? Or are they different things?